vaadin-time-picker-flow-parent/vaadin-time-picker-flow/src/main/resources/META-INF/resources/frontend/vaadin-time-picker/timepickerConnector.js

```diff
@@ -8,6 +8,7 @@ import {
   getSeparator,
   searchAmOrPmToken
 } from './helpers.js';
+import { TimePicker } from '@vaadin/time-picker';
```

Comment:
I wonder if we should import from the `src` folder?

Reply:
Updated.